`wasmi_wasi` cannot be built on Windows on Rust `nightly`

Thanks for #563 solving the problem, but a degraded `wiggle` introduces an old `cap-primitives`, which introduces more problems:

```
error[E0277]: the trait bound `file_type::FileType: std::sealed::Sealed` is not satisfied                                                                                                 
   --> C:\Users\Straw\.cargo\registry\src\github.com-1ecc6299db9ec823\cap-primitives-0.25.3\src\fs\file_type.rs:134:6
    |
134 | impl std::os::windows::fs::FileTypeExt for FileType {
    |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::sealed::Sealed` is not implemented for `file_type::FileType`
    |
    = help: the following other types implement trait `std::sealed::Sealed`:
              Child
              Command
              ExitCode
              ExitStatus
              ExitStatusError
              File
              OsStr
              OsString
            and 11 others
note: required by a bound in `std::os::windows::fs::FileTypeExt`
   --> D:\Straw\.rustup\toolchains\nightly-x86_64-pc-windows-msvc\lib/rustlib/src/rust\library\std\src\os\windows\fs.rs:507:24
    |
507 | pub trait FileTypeExt: Sealed {
    |                        ^^^^^^ required by this bound in `std::os::windows::fs::FileTypeExt`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `cap-primitives` due to previous error
```

--- a/crates/wasi/Cargo.toml
+++ b/crates/wasi/Cargo.toml
@@ -7,9 +7,9 @@ edition = "2021"
 
 [dependencies]
 
-wasi-common = "1.0.0"
-wasi-cap-std-sync = "1.0.0"
-wiggle = { version = "1.0.0", default-features = false, features = ["wiggle_metadata"] }
+wasi-common = "2.0"
+wasi-cap-std-sync = "2.0"
+wiggle = { version = "2.0", default-features = false, features = ["wiggle_metadata"] }
 wasmi = { version = "0.20.0", path = "../wasmi" }
 
 [dev-dependencies]

